### ISSUE TITLE
3.3 bug

### ISSUE THREAD

**Issue by yuanshengaoshou114** [NotThorny/AttackModifier#12]
Unexpected exception from an event executor:
java.lang.NoSuchMethodError: 'void emu.grasscutter.game.world.Scene.removeEntity(emu.grasscutter.game.entity.GameEntity, emu.grasscutter.net.proto.VisionTypeOuterClass$VisionType)'
        at thorny.grasscutters.AttackModifier.commands.AttackModifierCommand.removeGadgets(AttackModifierCommand.java:198)
        at thorny.grasscutters.AttackModifier.commands.AttackModifierCommand.addAttack(AttackModifierCommand.java:184)
        at thorny.grasscutters.AttackModifier.EventListener.onPacket(EventListener.java:38)
        at emu.grasscutter.plugin.PluginManager.invokeHandler(PluginManager.java:307)
        at emu.grasscutter.plugin.PluginManager.lambda$checkAndFilter$7(PluginManager.java:252)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
        at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
        at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
        at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
        at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:762)
        at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:276)
        at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
        at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1845)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
        at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
        at emu.grasscutter.plugin.PluginManager.checkAndFilter(PluginManager.java:252)
        at emu.grasscutter.plugin.PluginManager.lambda$invokeEvent$4(PluginManager.java:235)
        at java.base/java.lang.Iterable.forEach(Iterable.java:75)
        at emu.grasscutter.plugin.PluginManager.invokeEvent(PluginManager.java:235)
        at emu.grasscutter.server.event.Event.call(Event.java:30)
        at emu.grasscutter.server.game.GameServerPacketHandler.handle(GameServerPacketHandler.java:83)
        at emu.grasscutter.server.game.GameSession.handleReceive(GameSession.java:214)
        at emu.grasscutter.server.game.GameSessionManager$1.lambda$handleReceive$0(GameSessionManager.java:73)
        at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174)
        at io.netty.channel.DefaultEventLoop.run(DefaultEventLoop.java:54)
        at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.base/java.lang.Thread.run(Thread.java:833)

**Comment by yuanshengaoshou114** [comment]
Use
https://github.com/YuukiPS/DockerGS

**Comment by NotThorny** [comment]
Using AttackModifier v1.1? Although I don't use docker, I was able to compile it without issue using Yuuki's DockerGS, so I don't believe this is an issue that occurs with unmodified AttackModifier and DockerGS.

I've just went ahead and built it for you using DockerGS anyways, if you run into issues like this I'd recommend building with the jar you are using as the dependency instead of normal gc, since it differs from normal GC, which is what AttackModifier releases are built for.
[AttackModifier.zip](https://github.com/NotThorny/AttackModifier/files/10300871/AttackModifier.zip)
